src/client/components/components/Meal.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './Meal.css';
+
+function Meal({ meal }) {
+  return (
```

Comment:
Try destructuring for the object properties 💡

Reply:
Done! thank you again :)